src/screen/dashboard.rs

```diff
@@ -63,7 +64,7 @@ impl Dashboard {
             history: history::Manager::default(),
             last_changed: None,
             command_bar: None,
-            file_transfers: file_transfer::Manager::new(config.file_transfer.clone()),
+            file_transfers: file_transfer::Manager::new(config.file_transfer.clone(), proxy),
```

Comment:
I think file transfers should be aware of server level proxies as well if one is used over a root proxy?

Reply:
I didn't get what you mean, could you please provide me more hints about the change you are proposing?

Reply:
Maybe I'm misunderstanding something, see https://github.com/squidowl/halloy/pull/350#discussion_r1578126676

Reply:
I've fixed in efd7b54 and instead pass the proxy config in when new file transfer tasks are created. This is important because we don't want to hold onto a stale proxy config if the user refreshes their config while the app is running (which we now support).